app/src/sharedTest/java/org/oppia/android/app/home/RecentlyPlayedFragmentTest.kt

```diff
@@ -834,28 +813,28 @@ class RecentlyPlayedFragmentTest {
   fun testRecentlyPlayedTestActivity_checkSpanForItem3_spanSizeIsOne() {
     fakeOppiaClock.setFakeTimeMode(FakeOppiaClock.FakeTimeMode.MODE_UPTIME_MILLIS)
     storyProgressTestHelper.markRecentlyPlayedFractionsStory0Exp0(
-      profileId = profileId,
+      profileId,
       timestampOlderThanOneWeek = false
     )
     storyProgressTestHelper.markRecentlyPlayedRatiosStory0Exp0(
-      profileId = profileId,
+      profileId,
       timestampOlderThanOneWeek = true
     )
     ActivityScenario.launch<RecentlyPlayedActivity>(
       createRecentlyPlayedActivityIntent(
-        internalProfileId = internalProfileId
+        internalProfileId
       )
     ).use {
       testCoroutineDispatchers.runCurrent()
-      onView(withId(R.id.ongoing_story_recycler_view)).perform(
+      onView(withId(R.id.promoted_story_recycler_view)).perform(
         scrollToPosition<RecyclerView.ViewHolder>(
           3
         )
       )
-      onView(withId(R.id.ongoing_story_recycler_view)).check(
+      onView(withId(R.id.promoted_story_recycler_view)).check(
         hasGridItemCount(
           spanCount = 1,
-          position = 3
+          position = 1
```

Comment:
just for confirmation are we changing the position here correctly?

Reply:
its 3. I have mistakenly changed it to 1